### PR TITLE
Adapt lib/version_utils is_server to online medium

### PR DESCRIPTION
**Pull request by DrMullings**
So we schedule the test correctly for SLE15 SP2

- Related ticket: https://progress.opensuse.org/issues/59975

@Soulofdestiny @SergioAtSUSE

### CODE REVIEW

**Comment by DrMullings** [comment]
Verification: https://openqa.suse.de/tests/3628931

**Comment by DrMullings** [comment]
Verification for Online: https://openqa.suse.de/tests/3628954

**Comment by DrMullings** [comment]
We don't have that test suite for Full Flavor as I see it

File: lib/version_utils.pm [hunk 1]
```diff
@@ -417,8 +417,9 @@ sub is_server {
     return 1 if is_sles4migration();
     return 1 if get_var('FLAVOR', '') =~ /^Server/;
     return 1 if get_var('PUBLIC_CLOUD');
+    #return 1 if (check_var('FLAVOR', 'Full') || check_var('FLAVOR', 'Online'));
```
Comment (Soulofdestiny) [inline lib/version_utils.pm]: please remove comment

Comment (DrMullings) [reply lib/version_utils.pm]: And its gone

**Pull request merged**